src/utils/files/getRowsFromMeta.ts

```diff
@@ -88,13 +95,23 @@ export const getRowsFromMeta = (columns: any[], records: any[]): any[] => {
           break;
         }
         case 'dropdown': {
-          let value: any = get(data, column.field);
+          let value: any = get(data, column.field.split('.')[0]);
```

Comment:
@RenzoPrats 
I just think you overcomplexified the logic there, but the overall changes are good!